src/main/java/viewbot/ViewBot.java

```diff
@@ -84,7 +81,19 @@ private Runnable getExecutable() {
             }
             String ip = fullIp[0];
             int port = Integer.parseInt(fullIp[1]);
-            HttpClient httpClient = new HttpClient(ip, port);
+            HttpClient httpClient;
+            if (fullIp.length == 4) {
+                String user = fullIp[2];
+                String pass = fullIp[3];
+                httpClient = new HttpClient(ip, port, user, pass);
+                writeToLog("Configuring proxy with authentication!");
```

Comment:
This will spam log area. Let's write to log only when it's invalid proxy. (Imagine if you have 10k proxies)
![изображение](https://user-images.githubusercontent.com/51200522/121244053-1b6aa680-c8a7-11eb-9528-b9667f9b0aec.png)

Reply:
Removed the log